components/Home/Header/index.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from '../../../styles/Home.module.css'
+import { Base64Image } from '../../Core'
+
+interface HeaderProps {
+  name: string,
+  headline: string,
+  image: string
+}
+
+export default function Header({ name, headline, image }: HeaderProps) {
+  return (
+    <section className={styles.header}>
+      <Base64Image className={styles.profilePicture} image={image} alt='Profile picture' hideNull={false} />
+      <span className='primaryText'>{name}</span>
```

Comment:
consistency in css usage

Reply:
You mean local vs global styling? I actually think this quite consistent. Some are just global styling to be consistent with the styling of the rest of the app.

Reply:
Hmm well yeah, so className='primaryText' is coming from globals? I guess its ok. It just can get quite hard to maintain. I would personnally create separate Heading1, Heading2, Heading3 components and Paragraph component to reduce the need for Global styling.

Reply:
Hairy muff